src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineHelper.cs

```diff
@@ -349,13 +338,32 @@ public static void ExpandAncestors(ImmutableArray<DocumentSymbolUIItem> document
             }
         }
 
-        internal static void UnselectAll(IEnumerable<DocumentSymbolUIItem> documentSymbolItems)
+        internal static void UnselectAll(IEnumerable<DocumentSymbolDataViewModel> documentSymbolItems)
         {
             foreach (var documentSymbolItem in documentSymbolItems)
             {
                 documentSymbolItem.IsSelected = false;
                 UnselectAll(documentSymbolItem.Children);
             }
         }
+
+        internal static bool AreAllCollapsed(IEnumerable<DocumentSymbolDataViewModel> documentSymbolViewModelItems)
+        {
+            if (!documentSymbolViewModelItems.Any())
+            {
+                return false;
```

Comment:
consider doc'ing this.  it's unclear to me why having no iteams would mean you were not collapsed.